src/cio_websocket.c

```diff
@@ -149,19 +145,19 @@ static void send_close_frame(struct cio_websocket *ws, enum cio_websocket_status
 {
 	ws->close_status = cio_htobe16(status_code);
 	enum cio_error err = cio_timer_init(&ws->close_timer, ws->loop, NULL);
-	if (err == CIO_SUCCESS) {
+	if (likely(err == CIO_SUCCESS)) {
 		err = ws->close_timer.expires_from_now(&ws->close_timer, close_timeout_ns, close_timeout_handler, ws);
-		if (err == CIO_SUCCESS) {
+		if (likely(err == CIO_SUCCESS)) {
 			send_frame(ws, (uint8_t *)&ws->close_status, sizeof(ws->close_status), CIO_WEBSOCKET_CLOSE_FRAME, close_frame_written);
 			return;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this redundant jump. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=c%3AS3626)